### DISCUSSION TITLE
Where is the documentation and pdfs?

### DISCUSSION THREAD

**Discussion by jcbritobr** [IndySockets/Indy#584] [Q&A]
Notice that all links from old website are broken. Blog says repository is here now, but I cant find any documentation. May anyone help?

**Comment by jcbritobr** [comment]
Found some here.
http://mediaserver.fh-wedel.de/public/labp/indy/downloads.html
I think will be good get them as fast as possible.

**Comment by rlebeau** [comment]
As the [blog posting](https://www.indyproject.org/2021/02/10/links-to-old-indy-website-pages-are-currently-broken/) says, links to the old server are dead, and the content hosted on the old server has not been restored on the live server yet.  In the meantime, there is an archived copy of the old content on Archive.org:
https://web.archive.org/web/20201126071600/http://ww2.indyproject.org/index.en.aspx

**Comment by jcbritobr** [comment]
Yep, I read. But why not put the documentation on github repository?

**Comment by rlebeau** [comment]
Because I just have not had any time to put the files ANYWHERE yet.  Whether that is on GitHub, or on the website, or wherever.

**Comment by jcbritobr** [comment]
If you like I had time. I can help you to put the files in github.

**Comment by leus** [comment]
Remy, are you open to accept pull requests to port the web site to Jekyll? Also, is there a repository for the documentation?

**Comment by rlebeau** [comment]
> Remy, are you open to accept pull requests to port the web site to Jekyll?

I have no experience with Jekyll (and before this ticket, I'd never even heard of it).  The current site is hosted on WordPress, I don't see that changing anytime soon.

> Also, is there a repository for the documentation?

Not at this time.  It is on my TODO list.

**Comment by regs01** [comment]
CHM version, if anyone needs

[IndyDocs_10.1.5.0_HtmlHelp.zip](https://github.com/IndySockets/Indy/files/7782730/IndyDocs_10.1.5.0_HtmlHelp.zip)
